site/source/tutorials/003_creating_widgets/index.md

```diff
@@ -237,25 +243,94 @@ export default class WorkerContainer extends WorkerContainerBase<ThemeableProper
 
 Now we can update the `render` method to include some workers. Add the following to the top of the `render` method:
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerContainer.ts' lines:13-29 %}
+```ts
+		const workers: DNode[] = [
+			w(Worker, {
+				key: '1',
+				firstName: 'Tim',
+				lastName: 'Jones'
+			}),
+			w(Worker, {
+				key: '2',
+				firstName: 'Alicia',
+				lastName: 'Fitzgerald'
+			}),
+			w(Worker, {
+				key: '3',
+				firstName: 'Hans',
+				lastName: 'Mueller'
+			})
+		];
+```
 
 Notice that we have added an `key` property to each child. This is needed so that we can differentiate between the children. If you add multiple children that have the same tag name, e.g. `div` or widget name, e.g. `Worker`, then you will need to add a property that makes each child unique.
 In the code example shown above, we have added an `key` property and set the value to be unique for each child widget.
 
 We can now pass these workers as children to the container, by replacing the empty array in the `v` function's third argument with this array:
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerContainer.ts' lines:31-33 %}
+```ts
+		return v('div', {
+			classes: this.classes(css.container)
+		}, workers);
+```
 
 Now it is time to add styling rules for the `WorkerContainer`. Inside  of `styles/workerContainer.css`, add the following rule:
 
 {% include_codefile 'demo/finished/biz-e-corp/src/styles/workerContainer.css' lang:css %}
 
 Finally, let's update the `App` class to replace the `Worker` widget with the new `WorkerContainer`.
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' %}
+```ts
+	protected render(): DNode {
+		return v('div', [
+			w(Banner, {}),
+			w(WorkerContainer, {})
+		]);
+	}
+```
 
 The application now renders three workers in the `WorkerContainer` widget, allowing us to control how they are laid out without impacting the overall application.
 
+## Final steps - refactoring worker data to App
+
+When we created the `WorkerContainer`, we added the `Worker` data directly into it to ensure that it functioned properly. While this allowed us to develop the widget quickly, it will limit our ability to interact with that data since it is scoped to the `WorkerContainer` widget. In order to make the data more accessible, we need to move it up to the `App` widget. To do this, we will make two changes. First, we will move the `Worker` data up to the `App` widget and then we will update the `WorkerContainer` to receive it.
```

Comment:
"it will limit" while I know what it means here, it has the potential to throw off a novice user